homeassistant/components/reolink/config_flow.py

```diff
@@ -287,6 +287,7 @@ async def async_step_user(
             if not errors:
                 user_input[CONF_PORT] = host.api.port
                 user_input[CONF_USE_HTTPS] = host.api.use_https
+                user_input[CONF_PRIVACY] = host.api.supported(None, "privacy_mode")
```

Comment:
Shouldn't this key be better called `CONF_SUPPORTS_PRIVACY_MODE` or something, to better reflect that it's a cached value from the camera?

Reply:
Yes, I just wanted to be short.
But it makes sence, changed it.